packages/website/src/features/Docs/DocsCannonfilesPage.tsx

```diff
@@ -427,7 +438,20 @@ export const DocsCannonfilesPage: FC = () => {
                     #
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Metadata contains external information related to the cannon
+                  package. Currently metadata includes the following:
+                  <ul>- Git Repo URL</ul>
```

Comment:
this HTML does not appear correct...? how does it look in your browser? normally lists are represented by the format

```
<ul>
<li>One list item</li>
<li>Two list item</li>
</ul>
```

Reply:
There are chakra components for lists.